doc: add FAQ for orbit running alongside osquery (#3685)
fleetdm/fleet: docs/01-Using-Fleet/FAQ.md

## How does Fleet deal with IP duplication?

Fleet relies on UUIDs so any overlap with host IP addresses should not cause a problem. The only time this might be an issue is if you are running a query that involves a specific IP address that exists in multiple locations as it might return multiple results - [Fleet's teams feature](https://fleetdm.com/docs/using-fleet/teams) can be used to restrict queries to specific hosts.

> ## Can orbit run alongside osquery?

Reviewer: ```suggestion
## Can Orbit run alongside osquery?
```

Reviewer: One more typo, sorry @DominusKelvin. Uppercase "O" on Orbit. 👍🏻

> The Osquery instance that orbit runs uses its own database directory that is stored within the orbit directory. So orbit can run alongside plain osquery.

Reviewer: ```suggestion
Yes, Orbit can be run alongside osquery. The osquery instance that Orbit runs uses its own database directory that is stored within the Orbit directory.
```
